attesters/csv/collect_evidence.c

```diff
@@ -81,6 +82,19 @@ static int do_hypercall(unsigned int p1, unsigned long p2, unsigned long p3)
 	return (int)ret;
 }
 
+static size_t curl_writefunc_callback(void *contents, size_t size, size_t nmemb, void *userp)
```

Comment:
`curl_writefunc_callback` may called times, please refer to the usage of this in `sev-snp` to modify it.

Reply:
Do you mean the macro definition of CURL_RETRY_TIMES in sev-snp/collect_evidence.c，I fix the retry times as 5 in csv attester

Reply:
I mean you need to  refer to the implementation of `curl_writefunc_callback` in sev-snp/collect_evidence.c.